docs/source/api_reference/models.rst

```diff
@@ -555,6 +555,12 @@ TextCNNModel
 .. autoclass:: deepchem.models.torch_models.TextCNNModel
   :members:
 
+UNetModel
```

Comment:
Can you add a screenshot to show the docs render?

Reply:
Done